tests/ui/methods/missing-method-on-type-parameter.rs

```diff
@@ -0,0 +1,5 @@
+fn x<T: Copy>() {
```

Comment:
Suggestion: this looks like a regression test minimized from the ICE issue. Could we link to it here?